templates/compose-redbeams.tmpl

```diff
@@ -0,0 +1,44 @@
+{{{define "redbeams"}}}
+    redbeams:
+        environment:
+            - http_proxy={{{get . "HTTP_PROXY"}}}
+            - https_proxy={{{get . "HTTPS_PROXY"}}}
+            - CERT_VALIDATION
+            - REST_DEBUG
+            - 'REDBEAMS_JAVA_OPTS={{{getEscaped . "REDBEAMS_JAVA_OPTS"}}}'
+            - REDBEAMS_HBM2DDL_STRATEGY
+            - "REDBEAMS_DB_PORT_5432_TCP_ADDR={{{get . "COMMON_DB"}}}"
+            - "REDBEAMS_DB_PORT_5432_TCP_PORT=5432"
+            - REDBEAMS_DB_ENV_USER
+            - REDBEAMS_DB_ENV_PASS
+            - REDBEAMS_DB_ENV_DB
+            - REDBEAMS_DB_ENV_SCHEMA
+            - REDBEAMS_CLIENT_ID={{{get . "UAA_REDBEAMS_ID"}}}
+            - REDBEAMS_HOSTNAME_RESOLUTION=public
+            - REDBEAMS_ADDRESS_RESOLVING_TIMEOUT
+            - "REDBEAMS_CLOUDBREAK_URL={{{get . "CLOUDBREAK_URL"}}}"
+            - REDBEAMS_SCHEMA_SCRIPTS_LOCATION
+            - REDBEAMS_SCHEMA_MIGRATION_AUTO
+            - REDBEAMS_INSTANCE_NODE_ID={{{get . "CB_INSTANCE_NODE_ID"}}}
+            - REDBEAMS_LOG_LEVEL
+        labels:
+            - traefik.port=8080
+            - traefik.frontend.rule=PathPrefix:/rdb/
```

Comment:
rdb or redbeams for the prefix?

Reply:
I'm shooting for "rdb". The other services have very short prefixes as well.